NGCHM/WebContent/javascript/UI-Manager.js

```diff
@@ -1427,7 +1427,7 @@
 	);
 	dvAction ('MoveDownPage',
 	    "Move the primary normal, horizontal, or restricted horizontal view down one page",
-	    (e, mapItem) => { mapItem.currentRow += mapItem.dataPerRow; }
+	    (e, mapItem) => { mapItem.currentRow += mapItem.dataPerCol; }
```

Comment:
Is dataPerCol correct, or a misnomer?  Isn't this saying it's moving up and down (rows in one page)?

Reply:
It's been this way since the beginning.  It's opposite to the way I think.

I guess it means "number of data items (i.e. rows) in one column of the view".

Reply:
Good to know...

Reply:
I had thought changing dataPerCol and dataPerRow would have been a huge task, but it's actually only about 35 lines of code each, so manageable.

But it's now also in the save state information, so changing it would require a new map version and compatibility manager updates.

Reply:
We should discuss this, for a later PR.  A new map version is not needed if we "translate" the save state information on reading and writing.